src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -3070,490 +3115,551 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                                                                      lAlphaFieldBlanks,
                                                                      cAlphaFieldNames,
                                                                      cNumericFieldNames);
-            UtilityRoutines::IsNameEmpty(state, cAlphaArgs(1), cCurrentModuleObject, ErrorsFound);
-
-            state.dataHeatBal->CrossMixing(Loop).Name = cAlphaArgs(1);
-
-            state.dataHeatBal->CrossMixing(Loop).ZonePtr = UtilityRoutines::FindItemInList(cAlphaArgs(2), state.dataHeatBal->Zone);
-            if (state.dataHeatBal->CrossMixing(Loop).ZonePtr == 0) {
-                ShowSevereError(state,
-                                std::string{RoutineName} + cCurrentModuleObject + "=\"" + cAlphaArgs(1) + "\", invalid (not found) " +
-                                    cAlphaFieldNames(2) + "=\"" + cAlphaArgs(2) + "\".");
-                ErrorsFound = true;
-            }
+            // Create one Mixing instance for every space associated with this input object
+            auto &thisMixingInput = zoneCrossMixingObjects(mixingInputNum);
+            for (int Item1 = 1; Item1 <= thisMixingInput.numOfSpaces; ++Item1) {
+                ++mixingNum;
+                auto &thisMixing = state.dataHeatBal->CrossMixing(mixingNum);
+                thisMixing.Name = thisMixingInput.names(Item1);
+                int const spaceNum = thisMixingInput.spaceNums(Item1);
```

Comment:
Do you need the `spaceNum` and `zoneNum` temporaries? Are you doing anything except for saving them in `thisMixing`?

Reply:
Good point. Modified.